docs/OV_Runtime_UG/deployment/deployment-manager-tool.md

```diff
@@ -1,31 +1,30 @@
-# Deployment Manager {#openvino_docs_install_guides_deployment_manager_tool}
+# Deploying Your Application with Deployment Manager {#openvino_docs_install_guides_deployment_manager_tool}
 
-The Deployment Manager is a Python* command-line tool that creates a deployment package by assembling the model, IR files, your application, and associated dependencies into a runtime package for your target device. This tool is delivered within the Intel® Distribution of OpenVINO™ toolkit for Linux*, Windows* and macOS* release packages and is available after installation in the `<INSTALL_DIR>/tools/deployment_manager` directory.
+The OpenVINO™ Deployment Manager is a Python command-line tool that creates a deployment package by assembling the model, OpenVINO IR files, your application, and associated dependencies into a runtime package for your target device. This tool is delivered within the Intel® Distribution of OpenVINO™ toolkit for Linux, Windows and macOS release packages. It is available in the `<INSTALL_DIR>/tools/deployment_manager` directory after installation.
+
+This article provides instructions on how to create a package with Deployment Manager and then deploy the package to your target systems.
 
 ## Prerequisites
 
-* Intel® Distribution of OpenVINO™ toolkit
+To use the Deployment Manager tool, the following requirements need to be met:
+* Intel® Distribution of OpenVINO™ toolkit is installed. See the [Installation Guide](../../install_guides/installing-openvino-overview.md) for instructions on different operating systems.
 * To run inference on a target device other than CPU, device drivers must be pre-installed:
-   * **For Linux**, see the following sections in the [installation instructions for Linux](../../install_guides/installing-openvino-linux.md):
-     * Steps for [Intel® Processor Graphics (GPU)](../../install_guides/configurations-for-intel-gpu.md) section
+   * **For GPU**, see [Configurations for Intel® Processor Graphics (GPU)](../../install_guides/configurations-for-intel-gpu.md).
      * Steps for [Intel® Neural Compute Stick 2 section](../../install_guides/configurations-for-ncs2.md)
```

Comment:
@sgolebiewski-intel As I commented earlier, this part is not done updating. We need to categorize them by device, not OSes.

Reply:
It's been updated ;)